src/test/external-test/configure-gitea.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Retry gitea migrate until the db is ready
+timeout 30 bash -c 'until gitea migrate; do sleep 2; done'
+
+echo '==== BEGIN GITEA CONFIGURATION ===='
+
+function configure_admin_user() {
```

Comment:
not a big deal for this pr, but would love to explore any other (lightweight) alternatives to gitea for this test since we already test just for diversity of git servers

Reply:
Yeah it would have to be a push-to-create supporting service which we can look around for but to my knowledge is only gitea (and its derivatives) and gitlab right now.  @YrrepNoj may have found others though since he implemented the external stuff.

Reply:
👍